assets/css/_page/_single.scss

```diff
@@ -113,16 +113,16 @@
       }
     }
 
-    > h2 > .header-mark::before {
+    h2 > .header-mark::before {
       content: "#";
     }
-    > h3 > .header-mark::before {
```

Comment:
If the proposal will work, this would be the change:

```scss
> .outline-text-3 > h3 > .header-mark::before,
> h3 > .header-mark::before {
  /* ... */
}
```